conftest.py

```diff
@@ -14,21 +14,31 @@ def pytest_addoption(parser):
         default=False,
         help="Test additional MLIPs",
     )
+    parser.addoption(
```

Comment:
i think for this we shall use typer.

Reply:
Is it worth adding that now as a dependency when we only have these for pytest, or change it later when we start add CLI for the main code?

Reply:
happy with that but just put an issue about it.

Reply:
Cool, will do. It's also possible we can't change this one easily, as we don't explicitly import argparse here, but we can look into it when we add it elsewhere.